modules/nf-core/salsa2/meta.yml

```diff
@@ -0,0 +1,55 @@
+name: "salsa2"
+description: SALSA, A tool to scaffold long read assemblies with HiC
+keywords:
+  - assembly
+  - hi-c
+  - scaffolding
+  - long reads
+  - salsa
+  - salsa2
+tools:
+  - "salsa2":
+      description: "Salsa is a tool to scaffold long read assemblies with Hi-C."
+      homepage: "https://github.com/marbl/SALSA"
+      documentation: "https://github.com/marbl/SALSA"
+      tool_dev_url: "https://github.com/marbl/SALSA"
+      doi: "https://doi.org/10.1186/s12864-017-3879-z, https://doi.org/10.1371/journal.pcbi.1007273"
+      licence: "['MIT']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+  - fasta:
+      type: file
+      description: fasta file, Path to initial assembly, headers must not contain ':'
+      pattern: "*.{fa, fasta}"
+  - index:
+      type: file
+      description: ndex of the fasta file, Length of contigs at start
+      pattern: "*.{fa.fai, fasta.fai}"
+  - bed:
+      type: file
+      description: bed file, Bed file of alignments sorted by read names
+      pattern: "*.{bed}"
+
+
+output:
```

Comment:
`agp` emission needs to be listed in the output too.

Reply:
done